protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -119,347 +178,623 @@ service TradingDataService {
-    rpc ListEpochRewardSummaries(ListEpochRewardSummariesRequest) returns (ListEpochRewardSummariesResponse);
+    rpc ListEpochRewardSummaries(ListEpochRewardSummariesRequest) returns (ListEpochRewardSummariesResponse) {
+      option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+	tags: "Rewards";
+      };
+    }
 
-    // Rewards subscription
+    // Observe rewards
     //
     // Subscribe to a stream of rewards
-    rpc ObserveRewards(ObserveRewardsRequest) returns (stream ObserveRewardsResponse);
+    rpc ObserveRewards(ObserveRewardsRequest) returns (stream ObserveRewardsResponse) {
+      option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+	tags: "Rewards";
+      };
+    }
 
-    // Deposit
+    // Get deposit
     //
     // Get a deposit by its identifier
-    rpc GetDeposit(GetDepositRequest) returns (GetDepositResponse);
+    rpc GetDeposit(GetDepositRequest) returns (GetDepositResponse)  {
+      option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+	tags: "Funds";
+      };
+    }
 
-    // Deposits list
+    // List deposits
     //
     // Get a list of deposits for a given party
-    rpc ListDeposits(ListDepositsRequest) returns (ListDepositsResponse);
+    rpc ListDeposits(ListDepositsRequest) returns (ListDepositsResponse) {
+      option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+	tags: "Funds";
+      };
+    }
 
-    // Withdrawal
+    // Get withdrawal
     //
     // Get a withdrawal by its ID. Use the withdrawls list query to get withdrawal IDs
-    rpc GetWithdrawal(GetWithdrawalRequest) returns (GetWithdrawalResponse);
+    rpc GetWithdrawal(GetWithdrawalRequest) returns (GetWithdrawalResponse) {
+      option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+	tags: "Funds";
+      };
+    }
 
-    // Withdrawals list
+    // List withdrawals
     //
     // Get a list of withdrawals for a given party
-    rpc ListWithdrawals(ListWithdrawalsRequest) returns (ListWithdrawalsResponse);
+    rpc ListWithdrawals(ListWithdrawalsRequest) returns (ListWithdrawalsResponse) {
+      option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+	tags: "Funds";
+      };
+    }
 
-    // Asset
+    // Get asset
     //
     // Get a single asset using its ID. Use the assets list query to get an asset's ID
-    rpc GetAsset(GetAssetRequest) returns (GetAssetResponse);
+    rpc GetAsset(GetAssetRequest) returns (GetAssetResponse) {
+      option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+	tags: "Assets";
+      };
+    }
 
-    // Assets list
+    // List assets
     //
     // Get a list of assets using cursor based pagination
-    rpc ListAssets(ListAssetsRequest) returns (ListAssetsResponse);
+    rpc ListAssets(ListAssetsRequest) returns (ListAssetsResponse) {
+      option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+	tags: "Assets";
```

Comment:
oh there is an assets tag - what's the difference?

Reply:
addressed